### PR TITLE
TF-2533 [Part-4] File picker memory leak

### PR DESCRIPTION
**Pull request by tddang-linagora**
### Issue
- #2533 

### Demo
**Composer before**

https://github.com/linagora/tmail-flutter/assets/160106668/2f332d7b-3d51-42e5-b8ee-9a00c0356ffe

**Composer after**

https://github.com/linagora/tmail-flutter/assets/160106668/b8453acc-c7ba-44c3-b8af-2f2d93dcc311

**Identity edit before**

https://github.com/linagora/tmail-flutter/assets/160106668/6b9b8491-35ee-436e-bb9b-1686feab78ad

**Identity edit after**

https://github.com/linagora/tmail-flutter/assets/160106668/1602c32b-0a13-44c9-ab71-60f32b908a11

### CODE REVIEW

File: lib/features/identity_creator/presentation/identity_creator_controller.dart [hunk 1]
```diff
@@ -141,7 +142,7 @@ class IdentityCreatorController extends BaseController {
     inputNameIdentityController.dispose();
     inputBccIdentityController.dispose();
     scrollController.dispose();
-    richTextWebController.onClose();
+    richTextWebController.dispose();
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_controller.dart:145]: Please use `    richTextWebController.onClose();`

File: lib/features/identity_creator/presentation/identity_creator_controller.dart [hunk 2]
```diff
@@ -490,6 +491,8 @@ class IdentityCreatorController extends BaseController {
     if (context.mounted) {
       if (filePickerResult?.files.isNotEmpty == true) {
         final platformFile = filePickerResult!.files.first;
+        // __file_picker_web-file-input
+        html.querySelector('#__file_picker_web-file-input')?.children.clear();
```
Comment (dab246) [inline lib/features/identity_creator/presentation/identity_creator_controller.dart:495]: please remove it

File: lib/features/identity_creator/presentation/identity_creator_controller.dart (hunk 1 above)
Comment (chibenwa) [reply lib/features/identity_creator/presentation/identity_creator_controller.dart]: Why?

File: lib/features/identity_creator/presentation/identity_creator_controller.dart (hunk 2 above)
Comment (chibenwa) [reply lib/features/identity_creator/presentation/identity_creator_controller.dart]: Why?

File: lib/features/identity_creator/presentation/identity_creator_controller.dart (hunk 1 above)
Comment (dab246) [reply lib/features/identity_creator/presentation/identity_creator_controller.dart]: > Why?

Who are you asking? What are you asking about?

File: lib/features/identity_creator/presentation/identity_creator_controller.dart (hunk 2 above)
Comment (dab246) [reply lib/features/identity_creator/presentation/identity_creator_controller.dart]: > Why?

Who are you asking? What are you asking about?

Comment (hoangdat) [reply lib/features/identity_creator/presentation/identity_creator_controller.dart]: use for testing. Removed

File: pubspec.lock [hunk 3]
```diff
@@ -497,13 +497,14 @@ packages:
...
     description:
-      name: file_picker
-      sha256: "9d6e95ec73abbd31ec54d0e0df8a961017e165aba1395e462e5b31ea0c165daf"
-      url: "https://pub.dev"
-    source: hosted
-    version: "5.3.1"
+      path: "."
+      ref: master
+      resolved-ref: "6e9bd03c6884865f6958bdb86199c7dfaf459963"
+      url: "https://github.com/tddang-linagora/flutter_file_picker"
```
Comment (hoangdat) [inline pubspec.lock:505]: use linagora repo while waiting for response to upstream's PR

File: pubspec.yaml [hunk 4]
```diff
@@ -261,6 +259,13 @@ dependency_overrides:
 
   firebase_core_platform_interface: 4.6.0
 
+  file_picker:
```
Comment (hoangdat) [inline pubspec.yaml:262]: add `Todo` comment to have information about upstream's PR

File: pubspec.lock (hunk 3 above)
Comment (chibenwa) [inline pubspec.lock:505]: If as a comment you can put the link to the upstream PR then that is even better!

**Review by hoangdat — APPROVED** [review]

**Comment by dab246** [comment]
- [ ] Rebase & Squash commit

**Review by dab246 — APPROVED** [review]

**Review by hoangdat — APPROVED** [review]

**Pull request merged**